controllers/observability_reconciler.go

```diff
@@ -344,20 +379,32 @@ func (r *ObservabilityReconciler) Reconcile(ctx context.Context, _ []controller.
 	logger.V(1).Info("observability enabled, creating monitors")
 
 	// Kuadrant Operator monitor
-	kOpMonitorSpec.SetNamespace(r.namespace)
-	r.createMonitor(ctx, monitorObjs, kOpMonitorSpec, observability.ServiceMonitorsResource, logger)
+	kOpMonitor := kOpMonitorBuild(r.namespace)
+	err := r.createServiceMonitor(ctx, kOpMonitor, logger)
+	if err != nil {
```

Comment:
The errors are deliberately only logged, not propagated up the stack, to allow other monitors to possibly be still created.

Reply:
Ok, I will leave as it is then. However, if there is a net issue or API server downtime issue, the operator would not retry until there is another event :shrug: